app/drivers/adc.c

```diff
@@ -1,7 +1,22 @@
+#include <stdbool.h>
+
 #include "drivers/gpio.h"
 #include "drivers/adc.h"
 
+#define NUM_OF_CHANNELS 2
+
 ADC_HandleTypeDef adc_handle;
```

Comment:
this should have a m_ prefix too but let's skip and fix once we introduce the module struct.